src/Ng/CommandHelpers.cs

```diff
@@ -119,7 +119,8 @@ public static StorageFactory CreateStorageFactory(IDictionary<string, string> ar
                 { Arguments.StorageKeyValue, Arguments.StorageKeyValue },
                 { Arguments.StorageContainer, Arguments.StorageContainer },
                 { Arguments.StoragePath, Arguments.StoragePath },
-                { Arguments.StorageSuffix, Arguments.StorageSuffix }
+                { Arguments.StorageSuffix, Arguments.StorageSuffix },
+                { Arguments.StorageOperationMaxExecutionTimeInSeconds, Arguments.StorageOperationMaxExecutionTimeInSeconds },
```

Comment:
Does this really need to be a parameter? If we make this a simple const in the code, I think it will be a lot easier. Asked another way, when would one environment need a different timeout than another. Do we have HTTP client timeouts set anywhere else on server side?

If it must be a parameter can we default it to a sane value then make all of the nullable ints `int?` into just `int`?